test/AxelarGateway.js

```diff
@@ -2215,12 +2152,12 @@ describe('AxelarGateway', () => {
         });
 
         it('should return correct value for adminThreshold', async () => {
-            const epoch = 1;
+            const epoch = getRandomInt(Number.MAX_SAFE_INTEGER);
             expect(await gateway.adminThreshold(epoch)).to.eq(0);
```

Comment:
thought about this again, and maybe it's worth also explicitly verifying for the epoch = 1 case in both tests, since we always start with that.

Reply:
added epoch=1 and then a random number test for both tests